Automattic/newspack-blocks
# feat: add iframe block

add an iframe block that will let publishers to embed an iframe either by its URL or by uploading
the iframe archive

### All Submissions:






Add an iframe block to let publishers embed iframes by adding the iframes URL or by uploading the iframes content.



### How to test the changes in this Pull Request:

1. Pull the branch related to this PR.
2. Add a new post.
3. In the new post add the iframe block.
4. Add the iframe source URL and click on the embed button.
5. Observe the iframe embed in the post.
6. If set to full-screen mode in the sidebar, preview the post and observe that the iframe is filling all the page.
7. On another post add again an iframe block, and this time upload a zip archive that contains the iframe files (HTML, CSS, images, ...).
8. Observe the embed iframe in the post.

### Other information:
